### PR TITLE
Add Runbox

### PR DESCRIPTION
**Pull request by Rosiak**
Add Runbox Solutions to the list.
"Secure" email provider.

### CODE REVIEW

**Comment by Carlgo11** [comment]
The build failed because the logo you used is to large.
Here's a compressed version.

File: _data/email.yml [hunk 1]
```diff
@@ -104,6 +104,12 @@ websites:
       twitter: riseupnet
       img: riseup.png
       tfa: No
+
+    - name: Runbox
+      url: https://runbox.com
+      twitter: runbox
```
Comment (Carlgo11) [inline _data/email.yml]: Please change the twitter handle to `Runbox`

**Comment by Rosiak** [comment]
@Carlgo11 
Thanks for the quick response and sorry for the errors.

**Comment by Carlgo11** [comment]
No worries @Rosiak :smile:
So I've marked your PR as `looks-good`. That means that it's on it way of being merged. When ever another 2factorauth member has verified that the PR indeed looks all good they'll merge it into the master branch.
When that's all done it might take a few days for the changes to show up on the live site. This is because we manually update the site after careful verification locally on our end.

More information about this process can be found on our [wiki](https://github.com/2factorauth/twofactorauth/wiki/FAQ_Merge).

Have a good one!
//@Carlgo11 :palm_tree:

**Pull request merged**